### PR TITLE
Grammar and spelling fixes

### PR DESCRIPTION
**Pull request by a-choi**
My first pull request here on GitHub so if I'm doing it wrong or if there is anything I can do better please feel free to criticize and point me in the right direction.

This pull request just addresses a couple spelling and grammar issues that I found while using Museeks, nothing too much.

### CODE REVIEW

File: src/js/components/Settings/SettingsLibrary.react.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class SettingsLibrary extends Component {
             <ButtonGroup>
                 <Button bsSize='small' disabled={ this.props.refreshingLibrary } onClick={ this.addFolders.bind(this) }>
                     <Icon name='plus' fixedWidth />
-                    Add folder(s)
```
Comment (YurySolovyov) [inline src/js/components/Settings/SettingsLibrary.react.js]: this can actually add more than one folder, so this is valid

**Comment by YurySolovyov** [comment]
One comment, otherwise LGTM, thanks

**Comment by KeitIG** [comment]
As Yuri mentioned, you can add multiple folders at once, so you can keep  the "(s)".

**Comment by a-choi** [comment]
Woops, messed around with git and ended up undoing every change I made even though I was only trying to get rid of one change. Hopefully fixed soon!

**Comment by KeitIG** [comment]
All good, thanks for this !

**Pull request merged**